premerge/gke_cluster/main.tf

```diff
@@ -62,6 +69,12 @@ resource "google_container_node_pool" "llvm_premerge_linux" {
     resource_labels = {
       "goog-gke-node-pool-provisioning-model" = "on-demand"
     }
+
+    # Enable workload identity federation for this pool so that we can access
+    # GCS buckets.
+    workload_metadata_config {
+      mode = "GKE_METADATA"
+    }
   }
 }
 
```

Comment:
Is there any particular reason why we're not adding this to the llvm_premerge_libcxx node pool?